src/platforms/common/enriching-error-data/user-feedback.mdx

```diff
@@ -4,6 +4,8 @@ sidebar_order: 5
 redirect_from:
   - /learn/user-feedback/
 description: "Learn more about the embeddable JavaScript widget, which requests and collects the user information when an event occurs. Sentry pairs the feedback with the original event, giving you additional insight into issues."
+notSupported: [dotnet, android, cocoa, native]
```

Comment:
you should confirm this syntax works fwiw.. probably does, ive just only used native yaml lists (e.g. like redirect_from)

also cocoa -> apple

Reply:
The syntax I confirmed by checking this: https://github.com/getsentry/sentry-docs/blob/6e56af9bfe52e97ff8bcb199e6cca508a1a17185/src/platforms/common/configuration/index.mdx#L3

WIll fix `apple`.

Reply:
ops, quotes